OpenRA.Game/Server/Server.cs

```diff
@@ -215,6 +215,9 @@ public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData, boo
 
 					delayedActions.PerformActions(0);
 
+					if (dedicated)
```

Comment:
It would be useful if we could get a `// PERF: Dedicated servers need to drain the action queue to remove references blocking the GC from cleaning up disposed objects` or similar to explain this.